Test/git-issues/git-issue-604.dfy

```diff
@@ -0,0 +1,13 @@
+// RUN: %dafny "%s" /compile:3 /compileTarget:cs > "%t"
+// RUN: %dafny "%s" /compile:3 /compileTarget:js > "%t"
+// RUN: %dafny "%s" /compile:3 /compileTarget:java > "%t"
+// RUN: %dafny "%s" /compile:3 /compileTarget:go > "%t"
```

Comment:
This causes the test to be run 4 times, each time overwriting the output of the previous. There is no reason to indication a target language for the compilation, since this example is designed to cause a verification error, which means the compiler is never. So, use only one line and omit the two flags.